dns-integrator-operator/CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+<!-- Remember to update this file for your charm -- replace <charm-name> with the appropriate name. -->
+
+# Contributing
+
+## Overview
+
+This document explains the processes and practices recommended for contributing enhancements to the <charm-name> charm.
```

Comment:
nitpicking: charm name?

Reply:
I removed the file, it was a placeholder